catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -38,10 +38,14 @@
                     label="Monitor in place" value="in_place"/>
         </AD>
 
+        <AD description="Optional: Blacklisted file pattens.  These are regular expressions used to exclude different file patterns from being ingested.  The default values are temporary Windows files (*.tmp) and Windows thumbnails (thumbs.db)."
+            name="Blacklisted File Patterns" id="blacklistedFilePatterns" required="false" type="String"
+            cardinality="100"
+            default=".*.tmp,thumbs.db"/>
```

Comment:
The Camel File Component docs say that the exclude property is case sensitive.  Isn't it "Thumbs.db" not "thumbs.db"?   The documentation is lacking here.  It looks like you might be able to make it case insensitive using 
`(?i)thumbs.db`